torch/_inductor/codegen/cpp_prefix.h

```diff
@@ -92,6 +93,18 @@ Welford<T> welford_combine(const Welford<T> &acc, T data) {
   return result;
 }
 
+// Refer to https://github.com/pytorch/pytorch/blob/b5b36cf0c4e1958f1ff25120f5d4beeef3288187/
+// aten/src/ATen/native/SharedReduceOps.h#L419-L445
+template <typename scalar_t>
+inline bool greater_or_nan(scalar_t a, scalar_t b, int64_t idx_a, int64_t idx_b) {
+  // TODO<Leslie> Should we include the header file of <ATen/native/SharedReduceOps.h> or copy the implementation
```

Comment:
Considering the implementation is not that complex, maybe directly implement them here is better than including another header file?

Reply:
Thanks for the comment and changed.